asv/asv.conf.json

```diff
@@ -0,0 +1,75 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "pywt",
+
+    // The project's homepage
+    "project_url": "https://github.com/PyWavelets/pywt/",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    //"repo": "https://github.com/PyWavelets/pywt.git",
+    "repo": "..",
+
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "tip" (for mercurial).
+    "branches": ["master"], // for git
+    // "branches": ["tip"],    // for mercurial
+
+    // The DVCS being used.  If not set, it will be automatically
+    // determined from "repo" by looking at the protocol in the URL
+    // (if remote), or by looking for special directories, such as
+    // ".git" (if local).
+    "dvcs": "git",
+
+    // The tool to use to create environments.  May be "conda",
+    // "virtualenv" or other value depending on the plugins in use.
+    // If missing or the empty string, the tool will be automatically
+    // determined by looking for tools on the PATH environment
+    // variable.
+    "environment_type": "virtualenv",
+
+    // the base URL to show a commit for the project.
+    "show_commit_url": "http://github.com/PyWavelets/pywt/commit/",
+
+    // The Pythons you'd like to test against.  If not provided, defaults
+    // to the current version of Python used to run `asv`.
+    "pythons": ["2.7", "3.4"],
```

Comment:
I'd remove this to start with, cuts down on runtime and build issues. The benchmarking results won't really depend on Python version in an interesting/significant way.